### DISCUSSION TITLE
Overflow of message text in sequence diagrams

### DISCUSSION THREAD

**Discussion by asbjornu** [plantuml/plantuml#793] [Q&A]
Is it possible to have the second message text `POST /psp/paymentorders { consumerProfileRef }` in the below diagram overflow the `Swedbank Pay` participant line?

<img width="1012" alt="sequence" src="https://user-images.githubusercontent.com/12283/142702729-7054f41e-2a1b-4a7b-bba2-45255e533837.png">

It seems like message text is able to overflow participant lines unless the participant is the last (right-most) in the diagram. This theory is supported by the `<script src=rel:view-paymentorder.href />` message text overflowing the `Merchant` participant line. Is this possible to adjust somehow?

**Comment by The-Lum** [comment]
Hello @asbjornu,

See similar pseudo request _[force space between participants]_ here:
- [QA-12381/varying-participant-padding](https://forum.plantuml.net/12381/varying-participant-padding)

Regards.

**Reply by asbjornu** [reply]
Yeah, being able to configure a constant space between the participants and then have the text overflow when it doesn't fit would also solve my problem.
